app/assets/javascripts/request_for_comments.js

```diff
@@ -116,6 +116,9 @@ $(document).on('turbo-migration:load', function () {
             <button class="action-edit btn btn-sm btn-warning">' + I18n.t('shared.edit') + '</button> \
             <button class="action-delete btn btn-sm btn-danger">' + I18n.t('shared.destroy') + '</button> \
           </div> \
+          <div class="text-warning' + (comment.reportable ? '' : ' d-none') + '"> \
```

Comment:
Originally I wanted to add this feature by utilizing the Turbo framework. However, the changes seemed a bit large, and I want to release this soon. Therefore, I thought it was not a good time for the Turbo migration.